Plugin Cubot/src/main/java/net/simon987/cubotplugin/event/DeathCountEvent.java

```diff
@@ -0,0 +1,29 @@
+package net.simon987.cubotplugin.event;
+
+import net.simon987.server.event.GameEvent;
+import net.simon987.server.game.objects.GameObject;
+
+public class DeathCountEvent extends GameEvent {
+
+    public DeathCountEvent(){
+
+    }
+
+    public DeathCountEvent(GameObject object, int c) {
+        setSource(object);
+        if(c>=0){

```

Comment:
Check https://github.com/simon987/Much-Assembly-Required/blob/master/CONTRIBUTING.md for styling rules.
I don't think it is strictly enforced, but some spacing here would make the core more readable.

Also it would note be much effort to change `c` into `count`, I would prefer a descriptive name over the single letter

Reply:
I removed the int c entirely since I found it wasn't being used